maro/rl_v3/training/algorithms/ddpg.py

```diff
@@ -182,16 +180,16 @@ def __init__(self, name: str, params: DDPGParams) -> None:
 
     async def build(self) -> None:
```

Comment:
I forgot to remove the async here because this is no longer a coroutine

Reply:
Can we merge ``build`` into ``__init__`` now since we are always creating ops in the trainer?

Reply:
Sounds good.